awl/mslider.cpp

```diff
@@ -267,5 +268,4 @@ void MeterSlider::mousePressEvent(QMouseEvent* ev)
             }
       VolSlider::mousePressEvent(ev);
       }
-}
-
+}
```

Comment:
ought to leave the newline here just as original file, so that the diff is cleaner.